addons/civilian/functions/fnc_isHouseNearby.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function checks if there is any house nearby.
+ *
+ * Arguments:
+ * 0: Position to check <POSITION>
+ *
+ * Return Value:
+ * 0: Result <BOOL>
+ *
+ * Example:
+ * [position player] call afsk_civilian_fnc_isHouseNearby
+ *
+ * Public: No
+ */
+
+params ["_position"];
+
+private _houses = nearestTerrainObjects [_position, ["House"], 50, false];
```

Comment:
Might need blacklisting of some classes, if I recall correctly some lamp posts etc. are of type House too.

Reply:
Good catch, but I think it should not be too big problem. If it is, I will try to fix this behaviour some way.